### ISSUE TITLE
Text elements should have a reasonable contrast ratio

### ISSUE THREAD

**Issue by mineshmshah** [azayneeva/my-website#2]
[Warning] Text elements should have a reasonable contrast ratio (5)
See AX_COLOR_01 for more information.
<h2>​I'm a UX designer​</h2>​
<h2>​About me​</h2>​
<h2>​Education & Work Experience​</h2>​
<h2>​Tools and skills​</h2>​
<h2>​Interests​</h2>​
